crates/torii/core/src/model.rs

```diff
@@ -17,6 +17,8 @@ pub struct ModelSQLReader {
     name: String,
     /// The class hash of the model
     class_hash: FieldElement,
+    /// The contract address of the model
+    contract_address: FieldElement,
```

Comment:
@broody here is a notable change on the database which directly bounce to grpc proto and graphql models. If you have a chance to give it a look I would appreciate that. :pray:

Reply:
Seems like minimum impact, great stuff! 

@gianalarcon and I can help updating graphql tests to cover arrays in another PR

Reply:
> Seems like minimum impact, great stuff!
> 
> @gianalarcon and I can help updating graphql tests to cover arrays in another PR

Thank you for your help on that, definitely was it was a too broad scope. @gianalarcon if you want to catch up to give you context, will be happy to do so!

Reply:
Sure, I would like give me a hand with these issue